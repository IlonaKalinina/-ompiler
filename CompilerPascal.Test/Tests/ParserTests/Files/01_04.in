type
a = string;
var
b : a;
begin
end.