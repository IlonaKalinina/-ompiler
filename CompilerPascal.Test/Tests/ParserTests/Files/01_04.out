program 
├───type
│    └─── =
│         ├─── a
│         └─── string
├───var
│    └─── :
│         ├─── b
│         └─── a
│              └─── string
│    
begin
└─── end
